### PR TITLE
Added keys to lists

### PR DESCRIPTION
**Pull request by QapaMcFly**

### CODE REVIEW

File: src/components/Guides/Guides.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ import * as S from './styles'
 import Guide from './Guide'
 
 const Guides = props => {
-  const guides = props.guides.map(guide => <Guide {...guide} />)
+  console.log(props)
```
Comment (BennettDixon) [inline src/components/Guides/Guides.js]: Looks good, but let's remove the extra console log before pushing to prod if we're done with it.

**Review by BennettDixon — CHANGES_REQUESTED** [review]
Thanks for the first PR! LGTM other than the console log leftover from debugging. Cheers!

File: src/components/Guides/Guides.js (hunk 1 above)
Comment (QapaMcFly) [reply src/components/Guides/Guides.js]: Should be fixed, thanks forgot about it :)

**Review by BennettDixon — APPROVED** [review]
Thanks!

**Pull request merged**